packages/table/examples/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+    "version": "2.0.0",
+    "compilerOptions": {
+        "baseUrl": ".",
+        "declaration": true,
+        "experimentalDecorators": true,
+        "jsx": "react",
+        "module": "commonjs",
+        "moduleResolution": "node",
+        "noFallthroughCasesInSwitch": true,
+        "noImplicitAny": true,
+        "noImplicitReturns": true,
+        "noUnusedLocals": true,
+        "noUnusedParameters": true,
+        "removeComments": false,
+        "sourceMap": false,
+        "stripInternal": true,
+        "target": "es5",
+        "paths": {
+            "@blueprintjs/core": ["../../core/dist/index"]
```

Comment:
this shouldn't be necessary as the core package will come from `node_modules`.

it _is_ necessary in the core examples where we want to use the package name instead of relative paths.